### DISCUSSION TITLE
cn_cnt port available?

### DISCUSSION THREAD

**Discussion by stefandier** [DomiStyle/esphome-panasonic-ac#127] [Q&A]
Hi,

we have several cs-eu12aky-8 AC units. I opened it up but couldn't identify the required port? Could someone have a look at the pics and tell me if I am overlooking something?
[https://drive.google.com/drive/folders/17f7msujo_vw5WHOx-DJ-kFhgy-Mf_n2e?usp=sharing](url)

Thanks a lot for your help

**Comment by breithack** [comment]
You might want to open the black cover in picture 1 and see what's under there

**Comment by stefandier** [comment]
Just noticed that I asked here, sorry for never getting back. Slipped my mind, too many projects going on at once.

See my [other post](https://github.com/DomiStyle/esphome-panasonic-ac/discussions/185) for an update, if you are interested :)
